packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -144,15 +151,48 @@ function WorklistDataGrid(props) {
     grid.current.rows[focusedRow.current].cells[focusedCol.current].tabIndex = -1;
     focusedRow.current = clickedCell.parentElement.rowIndex;
     focusedCol.current = clickedCell.cellIndex;
+    setSelectedRow(clickedCell.parentElement.rowIndex);
+    setSelectedCol(clickedCell.cellIndex);
```

Comment:
It looks like you are duplicating the logic used for the focusedRow and focusedColumn ref variables.

Reply:
I've removed the cell select functionality so this is resolved